Dockerfile

```diff
@@ -22,8 +22,8 @@ RUN set -x \
  && wget -O /usr/local/bin/composer https://getcomposer.org/download/1.7.2/composer.phar \
  && chmod +x /usr/local/bin/composer \
  && wget https://chromedriver.storage.googleapis.com/2.42/chromedriver_linux64.zip -O /tmp/chromedriver.zip \
- && echo unzip \
  && unzip /tmp/chromedriver.zip -d /usr/local/bin \
+ && ln -s /usr/local/bin/chromedriver /usr/local/bin/phantomjs \
```

Comment:
Is it really a good idea to "name" chromedriver phantomjs? I think this might remove some error messages where the existance of phantomjs is checked, but at the end it cannot be called anyway. Shouldn't we just install phantomjs if it is an actual dependency or remove that dependency altogether (as it is discontinued)?